lib/bin/balena-preload.js

```diff
@@ -99,7 +99,7 @@ const getBalenaSdk = async (opts) => {
 	const { path } = await tmp.dir({ unsafeCleanup: true });
 	balenaSdkOptions.dataDirectory = path;
 	// @ts-expect-error
-	const balena = balenaSdk(balenaSdkOptions);
+	const balena = balenaSdk.getSdk(balenaSdkOptions);
```

Comment:
I have seen that text too, so I guess that's an indication why we stopped testing standalone mode.

Reply:
I suggest deleting this file in a `Change-type: major` commit that also amended the README file accordingly. (And just in case we later learn that we shouldn't have done it, perhaps keep a copy of the Typescript version on your laptop for a few months.) :-)